tests/testsuite/cargo_information/pick_msrv_compatible_package_within_ws_and_use_msrv_from_ws/in/crate2/Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+name = "crate2"
+version = "0.0.0"
+rust-version = "1.0.0"
```

Comment:
We used this MSRV because we could not find the MSRV from the `in/crate1` package.